src/Form/Type/ImmutableTabsType.php

```diff
@@ -24,14 +24,18 @@
  *
  * @author Benoit de Jacobet <benoit.de-jacobet@ekino.com>
  */
-class ImmutableTabsType extends ImmutableArrayType
+class ImmutableTabsType
 {
-    /**
-     * {@inheritdoc}
-     */
+    private $immutableArrayType;
+
+    public function __construct(ImmutableArrayType $immutableArrayType)
+    {
+        $this->immutableArrayType = $immutableArrayType;
+    }
+
     public function configureOptions(OptionsResolver $resolver)
     {
-        parent::configureOptions($resolver);
+        $this->immutableArrayType->configureOptions($resolver);
```

Comment:
The normal way to "extend" a form type with Symfony is to use the virtual inheritance system with the `getParent()` method. Then this line won't be needed.

Reply:
I directly inherit from AbstractType mainly for semantic purpose (it is a form) but still make use of composition to use configureOptions here. @qsomazzi don't hesitate if I get something wrong